extension/content.css

```diff
@@ -2,6 +2,16 @@ readerease-sentence {
 	background-color: inherit !important;
 }
 
+/* Make these containers be on top of any other element, because some websites
+have headers with high z-indices.
+It's okay to be obnoxiously high because user can always deactivate the extension.
+For where the magic value came from, see
+https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index/25461690#25461690
+ */
+#readingDisplayContainer, #optionsButton, #uiContainer {
```

Comment:
no variables for css :( https://stackoverflow.com/a/47508/4143394

Reply:
Hahahaha

Reply:
(on the very high z index value; I like it)